gulpfile.babel.js

```diff
@@ -1,63 +1,32 @@
 'use strict';
 
-import gulp from 'gulp';
-import plumber from 'gulp-plumber';
-import stylus from 'gulp-stylus';
+const gulp = require('gulp-help')(require('gulp'));
```

Comment:
Esse é o ponto que disse, estou usando o [gulp-help](https://www.npmjs.com/package/gulp-help) para quem for abrir o projeto pela primeira vez possa ter um melhor overview das tasks.
Porém, não consegui deixar num padrão de import, por causa do jeito que ele funciona.
Dei uma pesquisada e não encontrei uma solução. :(